client/src/index.js

```diff
@@ -8,8 +8,9 @@ import { BrowserRouter as Router } from 'react-router-dom';
 ReactDOM.render(
   <React.StrictMode>
     <Router>
-      <App />
       <GlobalStyles></GlobalStyles>
+      <App />
```

Comment:
I think you could just use `<GlobalStyles />` instead of `<GlobalStyles></GlobalStyles>`